onnx/backend/test/case/node/softmaxcrossentropy.py

```diff
@@ -20,13 +20,15 @@ def softmaxcrossentropy(x, target, weight=None, reduction='mean'):  # type: igno
         N, C = input_shape
         neg_gather_element_input = np.zeros((N, ), dtype=np.float32)
         for i in range(N):
-            neg_gather_element_input[i] = -inp[i][target[i]]
+            if target[i] != ignore_index:
```

Comment:
Just want to double-check. Do you have some python code to compare pytorch and this reference implementation?

Reply:
Yes, here it is.
[sce_test.zip](https://github.com/onnx/onnx/files/4400389/sce_test.zip)

Reply:
thanks @lara-hdr for running these tests and validating them with PyTorch.